app/models/gibct_site_mapper.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Creates sitemips for gi-bill-comparison-tool
+class GibctSiteMapper
+  DEFAULT_HOST = 'https://www.vets.gov/gi-bill-comparison-tool'
+
+  def initialize(ping: true, default_host: DEFAULT_HOST, sitemaps_path: nil, verbose: true)
```

Comment:
Since we never override `DEFAULT_HOST`, won't we generate a sitemap in dev/staging that points to www.vets.gov, potentially introducing wrong links (e.g. if an institution is added in staging but not yet in prod). If we need this to be configurable we can add an env var that distinguishes the host prefix.

Reply:
Oh I see ping is false so google ought to never crawl sitemaps for dev/staging, so maybe this is okay.

Reply:
@patrickvinograd I think the thing to do is only ping to true in production by examining the `original_url` in the request to see if it contains 'www.vets.gov'
 Do you see any issue?